packages/eslint-plugin-testing-library-react/src/configs/recommended.json

```diff
@@ -0,0 +1,6 @@
+{
+  "plugins": ["testing-library-react"],
+  "rules": {
+    "testing-library-react/prefer-expect-query-by": "error"
```

Comment:
Wondering if we should start with it as a warning?

Reply:
This is the "recommended" preset, so I would prefer to have it as an error as we want to "enforce" some best practices. In the default preset, it's turned off (I think).